docs/Blog-Post-Submission-Guide.md

```diff
@@ -0,0 +1,86 @@
+# Blog Post Contributor Guide
+
+This section describes the guidelines for contributing new blog posts to the
+Kata Containers website.
+
+## Share your stories on the StarlingX Website
```

Comment:
```suggestion
## Share your stories on the Kata Containers Website
```

Reply:
I should've used the editor rather than my eyes to double check there's no copy-paste error! Good catch!